src/style.css

```diff
@@ -1 +1,65 @@
 /* Styles go here */
+
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  
+}
+
+.header__logo {
+  height: 40px;
+  width: 40px;
+  margin: 10px 0 0 50px;
+}
+
+.nav {
+  display: inline-block;
+  margin: auto 40px;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  background-color: #fff;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__item {
+  position: relative;
+  margin: 0 10px;
```

Comment:
use only margin-right instead of adding for both sides. 
Also last element shouldn't have margin